### DISCUSSION TITLE
Analysis of APK doesnt finish !

### DISCUSSION THREAD

**Discussion by AlizadehF** [Pithus/bazaar#47] [Q&A]
the analysis is still running, refresh this page in few minutes by pressing F5. Something seems to be broken? Contact us!
when i upload the apk file, the above message is seen . what do i do?

**Comment by U039b** [comment]
Hi!
Could you please provide the link of the pending report?

**Comment by AlizadehF** [comment]
Hello,

https://beta.pithus.org/report/08313184329cfbe87242b5df79c5b045a5d0a7437d0c14da70851857b9cbe0a9

On Tue, Aug 10, 2021 at 3:34 PM Esther ***@***.***> wrote:

> Hi!
> Could you please provide the link of the pending report?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Pithus/bazaar/discussions/47#discussioncomment-1153710>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARQE2GR3ZCDIBRSWH6XS2E3T4EBSLANCNFSM5BZOGFSQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Alizade Faeze

**Reply by U039b** [reply]
It seems that your report is totally fine.

**Comment by jvoisin** [comment]
And here as well: https://beta.pithus.org/report/09ea0d45f1c5b0447478559432f0106ec8a3f33d87848c6d8d1a47786130aa72

**Reply by U039b** [reply]
The MobSF analysis never ends Oo It is stuck decompiling the APK.

**Comment by licaon-kter** [comment]
Any news? https://beta.pithus.org/report/7b5995605c97d25f894b5c7b0f3498e0451535bbab0c7826a996a516a74995fa & https://beta.pithus.org/report/6c31b3b8d6e849bb5dd923c4e7d7bfd60378169ae85a3c71e1718014e8b3a9a4

**Reply by U039b** [reply]
Hi!
Not sure what happened but your 2 analyses are now completed.

**Reply by licaon-kter** [reply]
They were not when I posted above (after 20h), yet they were when I looked 1 min ago :)

**Reply by U039b** [reply]
Yep, I manually restarted some parts of the analysis for your 2 reports.

**Comment by ramazansancar** [comment]
- https://beta.pithus.org/report/bc22a9dd78a4bd8f3cdf107897493826e94d3e83854e99771f7dfce2d59c7320
- https://beta.pithus.org/report/eb9327c666aab842872a54718259a1db7aab7d3c700adab4d2abe14119fbbd8c
